src/classes/User.ts

```diff
@@ -24,31 +36,45 @@ export default class User {
 
         if (this.#apiKey) {
             this.#authorizationHeader = {
-                headers: {
-                    Authorization: `Bearer ${this.#apiKey}`,
-                },
+                Authorization: `Bearer ${this.#apiKey}`,
             };
         } else {
             this.#authorizationHeader = undefined;
         }
+
+        this.#configuration = new Configuration({
+            apiKey: this.#apiKey,
+            fetchApi: fetch as unknown as ConfigurationParameters['fetchApi'],
+            headers: this.#authorizationHeader,
+            middleware: [],
+        });
```

Comment:
It looks like the codegen can add the `Bearer ` prefix for us [here](https://github.com/passageidentity/passage-node/blob/cade744d6996197f6246e612d9c79376e748ad72/src/generated/apis/ApiKeysApi.ts#L98-L103), so this could potentially be:
```ts
        this.#configuration = new Configuration({
            accessToken: this.#apiKey,
            fetchApi: fetch as unknown as ConfigurationParameters['fetchApi'],
            middleware: [],
        });
```
& we just get rid of `#authorizationHeader`.